google-cloud-error_reporting/README.md

```diff
@@ -1,82 +1,95 @@
-Stackdriver Error Reporting API for Ruby
-=================================================
-
-google-cloud-error_reporting uses [Google API extensions][google-gax] to provide an
-easy-to-use client library for the [Stackdriver Error Reporting API][] (v1beta1) defined in the [googleapis][] git repository
-
-
-[googleapis]: https://github.com/googleapis/googleapis/tree/master/google/devtools/clouderrorreporting/v1beta1
-[google-gax]: https://github.com/googleapis/gax-ruby
-[Stackdriver Error Reporting API]: https://developers.google.com/apis-explorer/#p/clouderrorreporting/v1beta1/
+# google-cloud-error_reporting
+
+[Stackdriver Error Reporting](https://cloud.google.com/error-reporting/) counts,
+analyzes and aggregates the crashes in your running cloud services. A
```

Comment:
s/the crashes/errors raised/

I know the current text is verbatim from cloudsite, but for Ruby I want to allow for data that is not a "crash".